config/default.yml

```diff
@@ -1247,6 +1247,12 @@ Style/Next:
     - skip_modifier_ifs
     - always
 
+Style/NilComparison:
+  EnforcedStyle: predicate_method
+  SupportedStyles:
+    - predicate_method
+    - explicit_comparison
```

Comment:
Think this should be named `comparison_method` (or operator).

Reply:
I see we named this simply `predicate` and `comparison` in a similar cop (NumericComparison), so we should probably stick to this for consistency.